samples/runtime/autosubstitution/README.md

```diff
@@ -0,0 +1,41 @@
+---
+page_type: sample
+languages:
+- qsharp
+products:
+- qdk
+description: "This sample uses the `Microsoft.Quantum.AutoSubstitution` NuGet package to provide alternative operations based on the simulator that is used."
+---
+
+# Using the AutoSubstitution rewrite step
+
+The `Microsoft.Quantum.AutoSubstitution` NuGet package offers a rewrite step
+that can link an operation `Op` to a set of pairs of an alternative operations
+`AltOp` and a simulator `Sim`, such that `AltOp` is executed as a replacement
+for `Op` when being invoked in `Sim`.
+
+[This post](https://devblogs.microsoft.com/qsharp/) in the [Q#
```

Comment:
Generally, using phrases like "this post" for links is suboptimal for search indexing; if possible, describing the link in the link itself can help make the post more visible in search.